README.md

```diff
@@ -17,11 +17,10 @@ First add credentials to a .env file, which you won't commit. If you track this
 ```env
 // .env.production
 ALGOLIA_APP_ID=XXX
-ALGOLIA_API_KEY=XXX
```

Comment:
why was this removed?

Reply:
I don't know, I didn't remove it, I just did a rebase on the master... might be that my master was out of date when I did it.

Reply:
done.